### ISSUE TITLE
Problem at Polygon with holes Validation

### ISSUE THREAD

**Issue by danistrauss** [angular-ui/ui-leaflet#256]
In [Leaflet documentation]( http://leafletjs.com/reference.html#polygon) for Polygon they say " You can also create a polygon with holes by passing an array of arrays of latlngs". When i try doing this with ui leaflet i get the "Invalid data passed to the polygon path" Error.  I created a[ fiddle ](http://jsfiddle.net/he274bjh/) where you can add the polygon with holes via ui-leaflet(error) and via leaflet.

**Comment by nmccready** [comment]
Your problem seems to be leaflet related. Basically your working with raw leaflet to create the polygon (which we are not messing with). So it would seem you should debug/check with leaflet. Also, does this work without ui-leaflet in an example?

**Comment by danistrauss** [comment]
Yes Polygon with holes works in leaflet but not in ui-leaflet. The problem seems not leaflet related to me. Because the coordinates object works with default leaflet polygon creation but it doesn't pass ui-leaflets polygon validation.

**Comment by nmccready** [comment]
Ok, I will look at your fiddle in more detail.
